services/QuillLMS/app/controllers/teachers/units_controller.rb

```diff
@@ -418,6 +418,12 @@ def score_info
       'Intermediate Diagnostic'
     when Activity::ADVANCED_DIAGNOSTIC_ACTIVITY_ID
       'Advanced Diagnostic'
+    when Activity::ELL_STARTER_DIAGNOSTIC_ACTIVITY_ID
+      'ELL Starter Diagnostic'
+    when Activity::ELL_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID
+      'ELL Intermediate Diagnostic'
+    when Activity::ELL_ADVANCED_DIAGNOSTIC_ACTIVITY_ID
+      'ELL Advanced Diagnostic'
```

Comment:
This `case` clause is getting pretty big.  I wonder if it's not time to replace it with something a little more straightforward like a hash:
```ruby
activity_ids_to_names = {
  Activity::STARTER_DIAGNOSTIC_ACTIVITY_ID: 'Starter Diagnostic',
  ...
}
activity_id = record['activity_id']
activity_ids_to_names[activity_id] || record['activity_name']
```